### DISCUSSION TITLE
Need help to disable it

### DISCUSSION THREAD

**Discussion by BRANthaD0N** [nickcoutsos/keymap-editor#161] [General]
![image](https://github.com/nickcoutsos/keymap-editor/assets/104870093/acff35fb-4521-4c29-bdeb-47e08b008a74)

**Comment by BRANthaD0N** [comment]
I keep going to the spot but it now allowing me 
![image](https://github.com/nickcoutsos/keymap-editor/assets/104870093/8509cd5c-74cf-43d9-8379-f361d6d8474f)

**Reply by nickcoutsos** [reply]
You need to select the specific repositories you want to use with the app. In your case only the `All repositories` option is displayed because you don't have any right now.

Have you followed the [Installing ZMK](https://zmk.dev/docs/user-setup) documentation yet?

**Reply by nickcoutsos** [reply]
@BRANthaD0N any luck?

**Comment by BRANthaD0N** [comment]
Yeah I was able to get in. Thanks that worked for me.
